extensions/functions_arithmetic.yaml

```diff
@@ -175,7 +175,10 @@ scalar_functions:
         return: fp64
   -
     name: "divide"
-    description: "Divide one value by another. Partial values are truncated."
+    description: >
+      Divide x by y. In the case of integer division, partial values are truncated.
```

Comment:
Can we describe more precisely what "partial values are truncated" means?
For example, what would be the result of `1 / -3`?

Reply:
+1, we use "truncation" for rounding toward zero (in contrast with ceil for +infinity and floor for -infinity) but it bears repeating.

Reply:
I have not seen truncation defined before as anything other than "round towards zero", but adding a few extra words does no harm, so updated here.